### ISSUE TITLE
Custom fields have no border styling.

### ISSUE THREAD

**Issue by deviousgeek** [auth0/Lock.Android#454]
When a custom field is added, the border styling applied to the email / username / password fields (ones that are not custom) is not applied.

__Code snippet or sample project that reproduces the bug:__

Lock implementation:
```
CustomField fieldName = new CustomField(R.drawable.com_auth0_lock_ic_username, CustomField.FieldType.TYPE_NAME, "name", R.string.hint_name);
List<CustomField> fields = new ArrayList<>();
fields.add(fieldName);
mLock = Lock.newBuilder(mAuth0, new AuthenticationCallback() {...})
        .withSignUpFields(fields)
        .allowSignUp(true)
...
        .build(this);
```

Lock theme:
```
<style name="LockTheme" parent="Lock.Theme">
        <item name="Auth0.HeaderLogo">@drawable/ui_logo_app</item>
        <item name="Auth0.HeaderTitle">@string/log_in_text</item>
        <item name="Auth0.HeaderTitleColor">@color/md_grey_900</item>
        <item name="Auth0.HeaderBackground">@color/appPrimaryBand</item>
        <item name="Auth0.PrimaryColor">@color/appPrimaryDark</item>
        <item name="Auth0.DarkPrimaryColor">@color/appPrimaryDarker</item>
        <item name="android:windowLightStatusBar" tools:targetApi="m">true</item>
    </style>
```

__Screenshots when appropriate:__

![image](https://user-images.githubusercontent.com/4342988/36574789-71f0b5cc-189b-11e8-9a31-257d25253724.png)

__Lock version:__ `2.8.1.`
__Android version & Phone Model when appropriate:__ `Samsung Galaxy S7 (SM-G930F), Android 7.0.`
__Versions of Gradle, Android Plugin, Build tools & Android SDK:__

```
Gradle: 4.4.
Android Plugin: 3.0.1.
SDK & Build Tools: 26 / 26.0.3.

```

**Comment by lbalmaceda** [comment]
Thanks for such detailed report.
> Lock version: 2.3.0.

Is this issue happening on the latest version as well?

**Comment by deviousgeek** [comment]
Sorry, no idea what compelled me to put that in. I am using 2.8.1 (updated issue), and re-tested again just to be sure. :)

It is an issue on release `2.8.1`.

**Comment by lbalmaceda** [comment]
Perfect. I'll look into it this week. 👋

**Comment by lbalmaceda** [comment]
@deviousgeek This is fixed on `2.8.2`.

**Comment by deviousgeek** [comment]
Legend.
